gpu-aware-scheduling/docs/usage.md

```diff
@@ -0,0 +1,115 @@
+# Usage with NFD and GPU-plugin
+This document explains how to get GAS working together with NFD and the GPU-plugin.
+
+To begin with, it will help a lot if you have been successfully already using the GPU-plugin with some deployments. That means your HW and cluster is most likely fine with GAS also.
```

Comment:
'been successful already'. Link the GPU device plugin for context here maybe?

Reply:
ok